metadata.json

```diff
@@ -30,7 +30,7 @@
     },
     {
       "name": "katello-katello",
-      "version_requirement": ">= 6.1.0"
+      "version_requirement": "<= 16.0.0"
```

Comment:
@ekohl does this pinning not affect the dependencies used by the unit tests here?

Reply:
No, that's in `.fixtures.yml`.